src/js/view/captionsrenderer.js

```diff
@@ -10,9 +10,11 @@ define([
 
     var _defaults = {
         back: true,
-        backgroundOpacity: 100,
+        backgroundOpacity: 50,
         edgeStyle: null,
+        fontSize: 14,
```

Comment:
`fontSize` is expected to be a number ([ref](https://support.jwplayer.com/customer/portal/articles/1482067-styling-captions-for-fcc-compliance)).